README.md

```diff
@@ -50,17 +54,15 @@ def capitalize(text: List[str]) -> List[str]:
 
 If you put this code in a file named `simple.py` in your terminal's working directory, you can serve it locally using [Uvicorn](https://www.uvicorn.org), by invoking `uvicorn simple:app --reload`.
 
-### Tableau FastAPI extensions support a limited set of data model features
+### FastAPI Tableau extensions support a limited set of data model features
 
 FastAPI Tableau extensions have a few constraints compared to standard FastAPI apps. Some of these are due to the ways that Tableau sends data. We'll describe known limitations here, and how to work within them.
 
-**You cannot use Pydantic models to describe your endpoints' input parameters. Put the arguments in the function definition directly instead.** FastAPI Tableau currently only supports declaring inputs in a function definition, as shown in the examples on this page. We plan to support Pydantic models in the near future. However, these will be subject to the same type and complexity limitations as function arguments, described below.
-
-**All parameters that receive data from Tableau must be [list fields](https://fastapi.tiangolo.com/tutorial/body-nested-models/?h=list#list-fields-with-type-parameter).** Tableau sends its data in JSON lists in the request body, which FastAPI converts to equivalent Python types. Compatible types are `List[str]`, `List[float]`, `List[int]`, and `List[bool]`.
+**All parameters that receive data from Tableau must be [list fields](https://fastapi.tiangolo.com/tutorial/body-nested-models/?h=list#list-fields-with-type-parameter).** Tableau sends its data in JSON lists in the request body, which FastAPI converts to equivalent Python types. Compatible types are `List[str]`, `List[float]`, `List[int]`, and `List[bool]`. You can define these parameters as arguments to an endpoint function, or as fields in a Pydantic model.
 
-**For compatibility with Tableau, your endpoints should also return one of these compatible list types.** Tableau expects to get back data of the same length as the input data it sent.
+**Your endpoints must also return data in a compatible list types, and the data returned must be the same length as the inputs received.** This requirement is not enforced by FastAPI Tableau, but Tableau maps the data sent back by an extension into a table column.
 
-**You can add query parameters to your functions by declaring singular (non-`List`) arguments.** FastAPI assumes that any `bool`, `float`, `int`, or `str` variables are query parameters. You can use these in the `script` argument when calling the API from Tableau.
+**You can use query parameters in your functions by declaring singular (non-`List`) arguments.** FastAPI assumes that any `bool`, `float`, `int`, or `str` variables are query parameters. You can use these in the `script` argument when calling the API from Tableau.
```

Comment:
This is confusing since just a few paragraphs ago, you said all params _must_ be Lists..